src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OperatorsShouldHaveSymmetricalOverloads.Fixer.cs

```diff
@@ -16,15 +24,96 @@ public abstract class OperatorsShouldHaveSymmetricalOverloadsFixer : CodeFixProv
 
```

Comment:
Similarly `OperatorsShouldHaveSymmetricalOverloadsFixer` can be made sealed and the `CSharp` and `VB` specific fixers can be deleted